### ISSUE TITLE
[BUG]: The Lord of the Rings: The Return of the King [SLUS-20770]

### ISSUE THREAD

**Issue by Ost3ntati0us** [PCSX2/pcsx2#11687]
### Describe the Bug

Broken skybox on, "the southern gate" level.

### Reproduction Steps

Simply start the level, "the southern gate" and the broken skybox will be noticed immediately.

### Expected Behavior

A skybox that isn't broken.

### PCSX2 Revision

PCSX2 v2.1.74

### Operating System

Windows 11

### CPU

i9 10850K

### GPU

RTX 3080

### GS Settings

Default

### Emulation Settings

Default

### GS Window Screenshots

![The Lord of the Rings - The Return of the King_SLUS-20770_20240809192311](https://github.com/user-attachments/assets/352c35a0-66a7-405b-9ff6-08b2e9264866)

### Logs & Dumps

https://drive.google.com/file/d/16CMrdaACaqaqSWGxj36MYP5iQ6jV2k25/view?usp=drive_link

**Comment by MrCK1** [comment]
The GS dump doesn't seem to help here since it's the same in SW/HW. Have you tried rounding/clamping modes and/or verified this isn't a bug on real HW as well?

**Comment by Ost3ntati0us** [comment]
I have verified that it is not a bug on original hardware. I set VU1 Clamping Mode to extra + preserve sign and the skybox is now rendering correctly. 

![Desktop Screenshot 2024 08 09 - 20 03 11 36](https://github.com/user-attachments/assets/88c09a08-e2f0-4763-b8fd-364fe6e4b8ba)

This also fixed the "crack of doom" level not being textured correctly.

![Desktop Screenshot 2024 08 09 - 20 12 12 16](https://github.com/user-attachments/assets/5a7d0e1c-1e32-4d50-aeda-835040939a23)

![Desktop Screenshot 2024 08 09 - 20 12 57 80](https://github.com/user-attachments/assets/c1728716-da61-45f7-bdcf-a5fe39797686)

Thank you for the suggestion I was about to give up.
